tests/test_bgp_neighbor_af.rb

```diff
@@ -407,7 +555,7 @@ def max_prefix(af, dbg)
     af.max_prefix_set(limit, threshold, warning)
     assert_equal(limit, af.max_prefix_limit,
                  "Test 4a. #{dbg} Failed to set limit to '#{limit}'")
-    assert_equal(threshold, af.max_prefix_threshold,
+    assert_equal((platform == :nexus ? threshold : 75), af.max_prefix_threshold,
```

Comment:
I would have thought that threshold = nil would be the way to set 'no threshold'. If 'nil' means '75', what means 'none'?